charts/cardano-node-api/values.yaml

```diff
@@ -7,13 +9,40 @@ cardano_node:
 image:
   repository: ghcr.io/blinklabs-io/cardano-node-api
   tag: 0.7.0
-ingress:
-  enabled: false
-  host: node-api.preview.local
 replicaCount: 1
 resources: {}
 tolerations:
 - key: kubernetes.io/arch
   operator: Equal
   value: arm64
   effect: NoSchedule
+
+affinity: {}
+
+ingress:
+  enabled: false
+  host: node-api.preview.dmtr.host
+  ingressClassName: kong
+
+service:
+  enabled: false
+  annotations: {}
+  ports:
+    - name: grpc
+      port: 443
+      protocol: TCP
+      targetPort: 9090
+  type: LoadBalancer
+
+# Example of a ClusterIP service
+  # ports:
+  # - name: api
+  #   port: 8080
+  #   protocol: TCP
+  #   targetPort: 8080
+  # type: ClusterIP
+
+secret:
+  enabled: false
+  tlsKey: foo
+  tlsCrt: foo
```

Comment:
We should probably have these default to an empty string (or not be defined at all). The `foo` from my previous comment was just because I was being lazy typing :grin: